### DISCUSSION TITLE
New GTK 4.10 API

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#4096] [General]
@mikedld Just a heads-up in case you hadn't seen: https://blog.gtk.org/2022/10/30/on-deprecations/

**Comment by mikedld** [comment]
We generally don't use any deprecated functionality by disabling it the hard way: https://github.com/transmission/transmission/blob/bf17a3baf1adae06476e9a324e7751649933a420/gtk/CMakeLists.txt#L180-L193

If those changes will lead to new deprecation warnings, may need to relax this a bit, because there're currently no plans for dropping GTK+ 3 support (some distributions don't even provide GTK(MM) 4 packages yet).
